swaylock/main.c

```diff
@@ -292,6 +294,10 @@ int main(int argc, char **argv) {
 	surfaces = create_list();
 	registry = registry_poll();
 
+	if (!registry) {
+		sway_abort("Unable to connect to sway");
```

Comment:
Yeah, I have called it `wayland compositor` now.

`wl_display_connect` expects `XDG_RUNTIME_DIR` but I guess we could change that. That's a different PR though, right? :)